lib/fizzy/execute.cpp

```diff
@@ -254,46 +254,22 @@ inline constexpr T rotr(T lhs, T rhs) noexcept
     return (lhs >> k) | (lhs << (num_bits - k));
 }
 
-inline uint32_t clz32(uint32_t value) noexcept
-{
-    // NOTE: Wasm specifies this case, but C/C++ intrinsic leaves it as undefined.
```

Comment:
This comment could be useful in new implementation, too.

Reply:
You mean to say that C++20 defines that? Yeah, can do.

Reply:
Adding this to all:
```
// NOTE: C++ specifies this case, but intrinsic leaves it as undefined.
```